compiler/src/main/java/com/squareup/anvil/compiler/codegen/KotlinPoet.kt

```diff
@@ -204,58 +268,3 @@ internal fun FileSpec.writeToString(): String {
   }
   return stream.toString()
 }
-
-/**
- * Removes invalid imports that can be caused by [requireTypeName] and copies imports from the
- * source file. Copying imports is necessary for star imports.
- */
-internal fun String.replaceImports(clazz: KtClassOrObject): String {
```

Comment:
That workaround isn't need anymore. The test coverage is really high, that's why I'm confident in this new solution.